eng/common/scripts/Invoke-GitHub-API.ps1

```diff
@@ -0,0 +1,180 @@
+param(
+  $AuthToken
```

Comment:
Given you are trying to provide a library of functions we shouldn't pass in the AuthToken at this level it should be passed in as a parameter to the individual methods. We need to be able to pass different tokens for different calls and we also should try to avoid having this being in this scope.